netcore/gen-xunit-runner/Program.cs

```diff
@@ -379,7 +379,7 @@ static int Main (string[] args)
 
 				var block = ParseText<BlockSyntax> (case_template
 					.Replace ("#MSG#", msg)
-					.Replace ("#NRUN#", test.MemberDataMethod == null ? "1" : ((IEnumerable<object []>) test.MemberDataMethod.Invoke (null, null)).Count ().ToString ()));
+					.Replace ("#NRUN#", test.MemberDataMethod == null ? "1" : ((IEnumerable) test.MemberDataMethod.Invoke (null, null)).Cast<object> ().Count ().ToString ()));
```

Comment:
Fixes `Unhandled Exception: System.InvalidCastException: Unable to cast object of type '<CopyTo_Hidden_Data>d__19' to type 'System.Collections.Generic.IEnumerable`1[System.Object[]]'.`
when it's used for System.Collections.Tests